sw/host/tests/manuf/manuf_cp_test_lock/src/main.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This test performs a lifecycle transition from `TEST_UNLOCKED0` to `TEST_LOCKED0`.
+//!
+//! We also test provisioning a `TEST_UNLOCKED` being programmed into OTP.
```

Comment:
What do you mean by this?

Reply:
I just meant that programming the chip to OTP (provisioning) is done within the test via JTAG rather than preloaded into the OTP before the test starts.

Updated the wording so hopefully it makes sense now

Reply:
ah got it, thanks.